### PR TITLE
Correct statuses for certain CSS properties

### PR DESCRIPTION
**Pull request by lukewarlow**
#### 309ff24a9f51908bd7c7a16d4e942cb6d46b7939
<pre>
Correct statuses for certain CSS properties
<a href="https://bugs.webkit.org/show_bug.cgi?id=257766">https://bugs.webkit.org/show_bug.cgi?id=257766</a>

Reviewed by Tim Nguyen.

Adds missing &quot;in development&quot; status for scrollbar-width and scrollbar-gutter.

Also change various other properties to be stable.

* Source/WebCore/css/CSSProperties.json:

Canonical link: <a href="https://commits.webkit.org/264945@main">https://commits.webkit.org/264945@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/1e00ee0448a9a19140e702f033930b257f8ab0e8

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/9192 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/9474 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/9702 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/10850 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/9110 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/9201 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/11463 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/9442 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/11976 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/9340 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/10290 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/7976 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/11006 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/7586 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/8392 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/15856 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/8686 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/8539 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/11863 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/9038 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/7375 "2 failures") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/8264 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/12488 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1054 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/8796 "Built successfully") | | |

### CODE REVIEW

File: Source/WebCore/css/CSSProperties.json [hunk 1]
```diff
@@ -4960,7 +4960,7 @@
                 "settings-flag": "overscrollBehaviorEnabled"
             },
             "status": {
-                "status": "in development"
+                "status": "experimental"
```
Comment (smfr) [inline Source/WebCore/css/CSSProperties.json]: stable

File: Source/WebCore/css/CSSProperties.json [hunk 2]
```diff
@@ -4982,7 +4982,7 @@
                 "parser-exported": true
             },
             "status": {
-                "status": "in development"
+                "status": "experimental"
```
Comment (smfr) [inline Source/WebCore/css/CSSProperties.json]: stable

File: Source/WebCore/css/CSSProperties.json [hunk 3]
```diff
@@ -5004,7 +5004,7 @@
...
```
Comment (smfr) [inline Source/WebCore/css/CSSProperties.json]: stable

File: Source/WebCore/css/CSSProperties.json [hunk 4]
```diff
@@ -5026,7 +5026,7 @@
                 "parser-grammar": "<<values>>"
             },
             "status": {
-                "status": "in development"
+                "status": "experimental"
```
Comment (smfr) [inline Source/WebCore/css/CSSProperties.json]: stable

File: Source/WebCore/css/CSSProperties.json [hunk 5]
```diff
@@ -5048,7 +5048,7 @@
...
```
Comment (smfr) [inline Source/WebCore/css/CSSProperties.json]: stable

**Review by smfr — CHANGES_REQUESTED** [review]

File: Source/WebCore/css/CSSProperties.json (hunk 1 above)
Comment (lukewarlow) [reply Source/WebCore/css/CSSProperties.json]: What's the difference between experimental and stable? I assumed experimental was when it still has a flag? If that's incorrect then I'll change some others too.

**Review by nt1m — APPROVED** [review]

**Pull request merged**